### DISCUSSION TITLE
Allow using English as alias

### DISCUSSION THREAD

**Discussion by wensimehrp** [OpenTTD/OpenTTD#11876] [Features]
The current OpenTTD search method only displays the result that exactly matches the filter, which, in some cases, it is not efficient. As a player who uses Chinese as my primary language, whenever I want to search for a cargotype or a setting, I have to switch my input method from none to Chinese, and the switch of input method means that I cannot use shortcuts anymore (most of them)

Another reason for this feature is that translated texts are often have inconsistancies, while in English they are the same pattern or word, for example, "vehicles" have multiple versions of Chinese translations. It could be "运输工具", "载具" or "车辆", but they all meant the same thing.

**Comment by PeterN** [comment]
If anyone wants to take a stab at this I think it would be quite an... interesting... task. More than one language pack would need to be loaded, and the string system would need to be adapted to allow using either one at a time, Care would need to be taken not to trash string caches as well.

Anywhere where search functionality I implemented would need be able to switch between loaded languages behind the scenes as it goes.
